packages/genesys-spark-components/src/components/legacy/gux-modal-legacy/gux-modal-legacy.scss

```diff
@@ -29,7 +29,10 @@
 @use '~genesys-spark/dist/scss/typography.scss';
 
 ::slotted(button:active:enabled) {
-  color: var(--gse-ui-link-active-foregroundColor);
+  color: var(
+    --gse-ui-link-active-foregroundColor,
+    --gse-ui-links-active-foregroundColor
+  );
```

Comment:
fallback is the same?

Reply:
@321gillian it's easy to miss, but there is an extra s in the fallback